src/ecdsa/test_pyecdsa.py

```diff
@@ -686,6 +686,16 @@ def test_VerifyingKey_encode_decode(curve, encoding):
     assert vk.pubkey.point == from_enc.pubkey.point
 
 
+def test_ecdh():
+    for vcurve in curves:
```

Comment:
please use pytest parametrize, like here:
https://github.com/warner/python-ecdsa/blob/4c92d31dec71c5fe478744d57d332d5ceb255a33/src/ecdsa/test_malformed_sigs.py#L35

then if one assert fails, the whole test won't be aborted